### PR TITLE
FEATURE: Mailing list mode default disabled

### PR DESCRIPTION
**Pull request by jomaxro**
Mailing list mode can generate significant email volume, especially on sites with a large userbase. Disable mailing list mode via site settings by default so sites don't experience an unexpectedly large cost from outgoing email.

### CODE REVIEW

**Review by justindirose — COMMENTED** [review]
Question here - would changing a default change the setting value on existing sites? I'm going to guess no, but I can't recall off the top of my head.

Otherwise, it's a reasonable change.

**Comment by jomaxro** [comment]
Yes, it will change the setting for sites that haven't modified it. This will have the effect of disabling mailing list mode on many sites, will make an announcement on Meta about this once merged.

**Review by eviltrout — APPROVED** [review]

**Comment by eviltrout** [comment]
Code is fine, I'll leave it to you to sort out the other details :)

**Pull request merged**
